team_comm:
  ros__parameters:
    # UDP broadcast address is the highest IP in the subnet e.g. 172.20.255.255
    # Sets local mode if set to loopback (127.0.0.1)
    target_ip: 10.87.255.255

    # Only used in non local mode with specific target_ip
    target_port: 3737
    receive_port: 3737

    # Only used in local mode on loopback
    # the team communication will bind to one of these ports and send to the other ports, depending on its bot_id
    local_target_ports:
      - 4001
      - 4002
      - 4003
      - 4004

    # Rate of published messages in Hz
    rate: 10

    # average walking speed in [m/s]
    avg_walking_speed: 0.2

    # lifetime of data [s]
    lifetime: 1

    # ROS Topics
    topics:
      team_data_topic: "team_data"
      strategy_topic: "strategy"
      time_to_ball_topic: "time_to_ball"
      pose_topic: "pose_with_covariance"
      ball_topic: "ball_position_relative_filtered"
      ball_velocity_topic: "ball_relative_movement"
      robots_topic: "robots_relative"
      cmd_vel_topic: "cmd_vel"
      gamestate_topic: "gamestate"
      move_base_goal_topic: "move_base/current_goal"

    # TF Frames
    map_frame: "map"
